docs/pages/setup/guides/teleport-operator.mdx

```diff
@@ -0,0 +1,194 @@
+kind: TeleportUser
+metadata:
+  name: myuser
+spec:
+  roles: ['myrole']
+```
+
+Apply the manifests to the Kubernetes cluster:
+
+```code
+$ kubectl apply -f teleport-resources.yaml
+```
+
+List the created Kubernetes resources:
+
+```code
+$ kubectl get teleportroles
+# NAME     AGE
+# myrole   10m
+
+$ kubectl get teleportusers
+# NAME     AGE
+# myuser   10m
+```
+
+Check the user `myuser` has been created in Teleport and has been granted the role `myrole`:
+```code
+$ PROXY_POD=$(kubectl get po -l app=teleport-cluster -o jsonpath='{.items[0].metadata.name}')
+$ kubectl exec -it "$PROXY_POD" -c teleport -- tctl users ls
+# User                          Roles
+# ----------------------------- -----------------------------
+# bot-teleport-operator-sidecar bot-teleport-operator-sidecar
+# myuser                        myrole
+```
+
+At this point the Teleport Kubernetes Operator is functional and Teleport users and roles can be managed from
+Kubernetes.
+
+## Going further
+
+### Exploring Teleport Kubernetes CRDs
+
+Available fields can be browsed with `kubectl explain` in a cluster with Teleport CRDs installed.
+For example the command:
+```code
+$ kubectl explain teleportroles.spec
+```
+Returns the following fields:
+```shell
+KIND:     TeleportRole
+VERSION:  resources.teleport.dev/v5
+
+RESOURCE: spec <Object>
+
+DESCRIPTION:
+    Role resource definition v5 from Teleport
+
+FIELDS:
+   allow	<Object>
+     Allow is the set of conditions evaluated to grant access.
+
+   deny	<Object>
+     Deny is the set of conditions evaluated to deny access. Deny takes priority
+     over allow.
+
+   options	<Object>
+     Options is for OpenSSH options like agent forwarding.
+```
+
+### Troubleshooting a resource not reconciled in Teleport
+
+During reconciliation the operator adds a `status` field on the Kubernetes resource.
+If Teleport returns some error it will be available in `status.conditions`.
+
+```code
+$ kubectl get teleportusers myuser -o yaml
+```
+
+For example, if a user has been granted a non-exising role the status will look like:
+
+```yaml
+apiVersion: resources.teleport.dev/v2
+kind: TeleportUser
+# [...]
+status:
+  conditions:
+  - lastTransitionTime: "2022-07-25T16:15:52Z"
+    message: Teleport resource has the Kubernetes origin label.
+    reason: OriginLabelMatching
+    status: "True"
+    type: TeleportResourceOwned
+  - lastTransitionTime: "2022-07-25T17:08:58Z"
+    message: 'Teleport returned the error: role my-non-existing-role is not found'
+    reason: TeleportError
+    status: "False"
+    type: SuccessfullyReconciled
+```
+
+Here `SuccessfullyReconciled` is `False` and the error is `role my-non-existing-role is not found`.
+
+If the status is not present or does not gives sufficient information to solve the issue, check the operator logs:
+
+```shell
+$ kubectl logs "$PROXY_POD" -c operator
+```
+
+<Admonition type="note">
+  In case of multi-replicas deployments, only one operator instance is active.
```

Comment:
What does "active" mean here? Also, this is the first time we mention leader election—there should at least be some explanation. And will the user need to check the logs of all Teleport Operator pods?

Reply:
Rephrased